py3status/modules/sysdata.py

```diff
@@ -10,19 +10,38 @@
         (default 75)
     med_threshold: percent to consider CPU or RAM usage as 'medium load'
         (default 40)
+    padding: length of space padding to use on the left
+        (default 0)
+    precision: precision of values
+        (default 2)
```

Comment:
I think we shouldn't have these.  Users can do all this in the format string eg `format = '{mem_used:05.2f}'`